website/content/en/status/report-2023-01-2023-03/aarch64-kasan.adoc

```diff
@@ -0,0 +1,23 @@
+=== Kernel Address Sanitizer on AArch64
+
+Contact: Kyle Evans <kevans@FreeBSD.org>
+
+Sanitizers are bug detection facilities which use a combination of instrumentation inserted by the compiler (LLVM in this case) and runtime state tracking to detect bugs in C code.
+They can automatically detect many types of C programming bugs, such as use-after-frees and uses of uninitialized variables, which may otherwise require substantial effort to identify.
+They are particularly effective in combination with regression testing suites or fuzzing tools such as link:https://github.com/google/syzkaller[syzkaller].
+Unlike tools such as Valgrind, software must be recompiled to enable a given sanitizer, but sanitizers can be used in the kernel.
+Kernels with sanitizers enabled incur a significant performance overhead from the runtime, in both CPU utilization and memory usage.
+
+As of gitref:89c52f9d59fa[repository=src], the kernel address sanitizer that was previously exclusive to amd64 has now been ported to arm64.
+
+Prior testing has been done on a decent variety of machines, including:
+
+- Various Ampere Altra machines
+- QEMU
+- Microsoft's "Volterra" Devkit
+- BHyve (WIP)
+
+Further testing on other hardware would be both welcomed and appreciated.
+
+Sponsor: Juniper Networks, Inc.
+Sponsor: Klara, Inc.
```

Comment:
We need to add a  `+` sign at the end of the first line to display both sponsors separately. This is a minor issue that can be fixed when pushing the commit.

```
Sponsor: Juniper Networks, Inc. +
Sponsor: Klara, Inc.
```

And thanks for your work!

Reply:
@dbaio: Can I assume this patch mentor approved once I fix the sponsor lines? Unless @kevans91 wants to commit it himself of course.

Reply:
yes, please go ahead.
and thanks again.